doc/EPIC.md

```diff
@@ -0,0 +1,110 @@
+
+### Data Plane
+The data plane operations for EPIC-HP path type packets are the same as for SCION path type packets, but the source additionally computes two per-packet validation fields for the penultimate and last ASes on the path. 
+The last two ASes need to validate the fields accordingly.
+A more concise description can be found in the EPIC-HP path type specification.
+
+## <a id="PathTypeFiltering"></a> Path Type Filtering
+Network operators should be able to clearly define which kind of traffic (SCION, EPIC-HP, EPIC-SAPV, and other protocols) they want to allow. 
+Therefore, for each AS and every interface pair, an AS can be configured with 1-bit flags to allow only certain types of traffic: 
+
+<p align=center>AllowedTraffic(If<sub>1</sub>, If<sub>2</sub>) = (flag<sub>SCION</sub>, flag<sub>EPIC-HP</sub>, flag<sub>COLIBRI</sub>, ...)</p>
+
+The order of the interfaces, (If<sub>1</sub>, If<sub>2</sub>) vs. (If<sub>2</sub>, If<sub>1</sub>), allows to enable and disable different types of traffic depending on the direction. 
+To exclusively allow SCION path type traffic (default) between interfaces 'x' and 'y' we would set:
+
+<p align=center>AllowedTraffic(x, y) = (1, 0, 0, ...)</p>
+
+And similarly to only allow EPIC-HP path type traffic:
+
+<p align=center>AllowedTraffic(x, y) = (0, 1, 0, ...)</p>
+
+## Best Practices
+There are two main applications for EPIC-HP:
+
+### <a id="HighlySecureHiddenPaths"></a> Highly Secure Hidden Paths
+The last and penultimate ASes on the hidden path only allow EPIC-HP traffic on the interface pairs that affect the hidden path. 
+With such a setup it is not possible for unauthorized sources to reach the services in the last AS. Therefore, EPIC-HP effectively prevents adversaries from running attacks like denial of service, or attack preparations like scanning the services for vulnerabilities. 
+
+If some host H1 inside an AS with such a setup wants to communicate with a host inside another AS that is also behind a hidden path, both hosts need to have valid authenticators to send traffic over the corresponding hidden paths. The hosts can exclusively communicate using EPIC-HP, and H1 needs to set the SCION-Response flag to zero.
+
+Note that hosts behind a hidden path can send SCION path type packets towards hosts in other ASes, but that those hosts can not send a response back if they do not have the necessary authenticators.
+
+### <a id="DOSSecureHiddenPaths"></a> DoS-Secure Hidden Paths
+The last and penultimate ASes on the hidden path generally allow EPIC-HP traffic, but prioritize it before any other traffic. 
+This means that DOS attacks are not possible, because an adversary is limited to sending low-priority SCION path type packets. However, an adversary can still reach the services behind the hidden path.
```

Comment:
Maybe mention the advantage that in this case the link can still be used as a "normal" access link.

Reply:
Added that now.